eslint-config/rules/plugin-jsdoc.js

```diff
@@ -0,0 +1,247 @@
+     */
+    'jsdoc/no-types': ['off'],
+
+
+    /**
+     * Checks that types in jsdoc comments are defined. This can be used to check unimported types.
+     */
+    'jsdoc/no-undefined-types': ['warn'],
+
+
+    /**
+     * Requires that block description, explicit `@description`, and `@param`/`@returns` tag descriptions are written in complete sentences
+     */
+    'jsdoc/require-description-complete-sentence': ['off'],
+
+
+    /**
+     * Requires that all functions have a description.
+     */
+    'jsdoc/require-description': ['off'],
+
+
+    /**
+     * Requires that all functions have examples.
+     */
+    'jsdoc/require-example': ['off'],
+
+
+    /**
+     * Checks that all files have a `@file`, `@fileoverview`, or `@overview` tag.
+     */
+    'jsdoc/require-file-overview': ['off'],
+
+
+    /**
+     * Requires a hyphen before the `@param` description.
+     *
+     * ಠ_ಠ
+     */
+    'jsdoc/require-hyphen-before-param-description': ['off'],
+
+
+    /**
+     * Checks for presence of jsdoc comments, on class declarations as well as functions.
+     */
+    'jsdoc/require-jsdoc': ['warn', {
+      require: {
+        FunctionDeclaration: true,
+        MethodDefinition: true,
+        ClassDeclaration: true,
+        ArrowFunctionExpression: false,
+        FunctionExpression: false,
+      },
+    }],
+
+
+    /**
+     * Requires that each `@param` tag has a `description` value.
+     */
+    'jsdoc/require-param-description': ['off'],
+
+
+    /**
+     * Requires that each `@param` tag has a `name` value
+     */
+    'jsdoc/require-param-name': ['warn'],
+
+
+    /**
+     * Requires that each `@param` tag has a `type` value.
+     */
+    'jsdoc/require-param-type': ['warn'],
+
+
+    /**
+     * Requires that all function parameters are documented.
+     */
+    'jsdoc/require-param': ['warn', {
+      exemptedBy: ['endpoint'],
+    }],
+
+
+    /**
+     * Requires that each `@property` tag has a `description` value.
+     */
+    'jsdoc/require-property-description': ['off'],
+
+
+    /**
+     * Requires that each `@property` tag has a `name` value.
+     */
+    'jsdoc/require-property-name': ['off'],
+
+
+    /**
+     * Requires that each `@property` tag has a `type` value.
+     */
+    'jsdoc/require-property-type': ['off'],
+
+
+    /**
+     * Requires that all `@typedef` and `@namespace` tags have `@property` when their type is a plain `object`, `Object`, or `PlainObject`.
+     */
+    'jsdoc/require-property': ['off'],
+
+
+    /**
+     * Requires a return statement in function body if a `@returns` tag is specified in jsdoc comment.
+     */
+    'jsdoc/require-returns-check': ['warn'],
+
+
+    /**
+     * Requires that the `@returns` tag has a `description` value.
+     */
+    'jsdoc/require-returns-description': ['off'],
+
+
+    /**
+     * Requires that the `@returns` tag has a `type` value.
+     */
+    'jsdoc/require-returns-type': ['off'],
```

Comment:
Same as above, could we warn against this as well for consistency?

Reply:
Agree